sklearn/feature_extraction/text.py

```diff
@@ -607,9 +607,10 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         A mapping of terms to feature indices.
 
     stop_words_ : set
-        Terms that were ignored because
-        they occurred in either too many
-        (`max_df`) or in too few (`min_df`) documents.
+        Terms that were ignored because they either
+        - occurred in too many documents (`max_df`)
+        - occurred in too few documents (`min_df`)
+        - were cut off by feature selection (`max_features`).
```

Comment:
@agramfort I *think* it doesn't render properly... 

Ref : From [scipy's documentation](http://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.sparse.linalg.eigs.html?highlight=magnitude)... A similar bulleted list in the documentation was changed to [this](https://github.com/scipy/scipy/commit/a36f106d84368d89da75c5877e7bac4d4a095026#diff-674d817ce47902732d15bc3861d70aa6R1132) and later to [this](https://github.com/scipy/scipy/commit/e37131be0489729a5adaead0a4a59c1c86745046#diff-674d817ce47902732d15bc3861d70aa6L1129)...